src/themes/default/components/core/blocks/Reviews/Reviews.vue

```diff
@@ -119,10 +119,17 @@ export default {
       }
     }
   },
-  computed: {
-    product () {
-      return this.$store.state.product
+  props: {
+    originalProduct: {
+      type: Number,
+      required: true
     },
+    currentProduct: {
```

Comment:
It would be better to pass the whole product object and original product object OR rather rename this property to `originalProductId` - otherwise is missleading

Reply:
Renaming would be the way to go, there is no need for Objects within Reviews.vue